packages/onboarding/src/utils.ts

```diff
@@ -1,6 +1,14 @@
 export const NEWSLETTER_FLOW = 'newsletter';
 export const LINK_IN_BIO_FLOW = 'link-in-bio';
+export const VIDEOPRESS_FLOW = 'videopress';
 
 export const isNewsletterOrLinkInBioFlow = ( flowName: string | null ) => {
 	return Boolean( flowName && [ NEWSLETTER_FLOW, LINK_IN_BIO_FLOW ].includes( flowName ) );
 };
+
+export const isTailoredSignupFlow = ( flowName: string | null ) => {
+	return Boolean(
+		flowName &&
+			( isNewsletterOrLinkInBioFlow( flowName ) || [ VIDEOPRESS_FLOW ].includes( flowName ) )
```

Comment:
Is there a reason for creating an array?

```suggestion
			( isNewsletterOrLinkInBioFlow( flowName ) || VIDEOPRESS_FLOW === flowName )
```

Reply:
That was half me not refactoring all the way, and half me thinking "what if someone wants to add a new item to the list"

But I guess we can let that person change it to an array if they need it.